examples/weather/main/weather.cpp

```diff
@@ -0,0 +1,1392 @@
+/* Simple firmware for a ESP32 displaying a static image on an EPaper Screen.
+ *
+ * Write an image into a header file using a 3...2...1...0 format per pixel,
+ * for 4 bits color (16 colors - well, greys.) MSB first.  At 80 MHz, screen
+ * clears execute in 1.075 seconds and images are drawn in 1.531 seconds.
+ */
+#define ESP32
+
+#include "esp_heap_caps.h"
+#include "esp_log.h"
+#include "esp_timer.h"
+#include "esp_types.h"
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+#include "sdkconfig.h"
+#include <stdio.h>
+#include <string.h>
+
+#include "epd_driver.h"
+
+#include "Arduino.h"
+#define ARDUINOJSON_ENABLE_ARDUINO_STRING 1
+#include <ArduinoJson.h>
+#include <HttpClient.h>
```

Comment:
This library is called HTTPClient for me... Maybe a different version? (Using arduino-esp for IDF 4.0)

Reply:
both are fine for me, changed